lib/node.js

```diff
@@ -117,7 +117,10 @@ declare class Buffer extends Uint8Array {
 }
 
 declare module "buffer" {
+  declare var Buffer: Buffer;
```

Comment:
this says that `require('buffer').Buffer` is an instance of a buffer, I think you'd want `...: Class<Buffer>`.

but since `Buffer` is a global, I don't think we need this one. I admit it's opinionated to say not to use `require('buffer').Buffer` but it seems to cause a minor headache for flow because it thinks the `source: Buffer` annotation below is a value since it shadows the name.